spinalcordtoolbox/reports/qc2.py

```diff
@@ -343,6 +345,146 @@ def sct_deepseg(
+                    else:
+                        axes[idx_row, idx_col].set_title(f'Sagittal slice #{sagittal_slice}')
+
+                # Add title to each row, (i.e., y-axis)
+                if idx_col == 0:
+                    axes[idx_row, idx_col].set_ylabel(f'Lesion #{lesion_label}\n'
+                                                      f'Inferior-Superior')
+                else:
+                    axes[idx_row, idx_col].set_ylabel('Inferior-Superior')
+
+                # Add x-axis label
+                axes[idx_row, idx_col].set_xlabel('Posterior-Anterior')
+
+                # Crop the slice around the lesion (to zoom in)
+                axes[idx_row, idx_col].set_xlim(np.min(np.where(im_label_data_cur)[1]) - 20,
+                                                np.max(np.where(im_label_data_cur)[1]) + 20)
+                axes[idx_row, idx_col].set_ylim(np.min(np.where(im_label_data_cur)[2]) - 20,
+                                                np.max(np.where(im_label_data_cur)[2]) + 20)
+
+                # --------------------------------------
+                # Add text for tissue bridges
+                # --------------------------------------
+                # Check if the [idx_row][sagittal_slice, 'dorsal'] key exists in the tissue_bridges_plotting_data
+                # If the key exists, it means that we have tissue bridges for the current lesion and sagittal slice
+                if idx_row < len(measure_pd):
+                    col_name_dorsal = f"slice_{int(sagittal_slice)}_dorsal_bridge_width [mm]"
+                    dorsal_bridge_width_mm = measure_pd[col_name_dorsal][idx_row]
+                    if col_name_dorsal in measure_pd.columns and not pd.isna(dorsal_bridge_width_mm):
+
+                        axes[idx_row, idx_col].text(min(np.where(slice_lesion)[0]) - 3,
+                                                    min(np.where(slice_lesion)[1]),
+                                                    f'Dorsal bridge\n{np.round(dorsal_bridge_width_mm, 2)} mm',
+                                                    color='red', fontsize=12, ha='right', va='bottom')
+
+                    col_name_ventral = f"slice_{int(sagittal_slice)}_ventral_bridge_width [mm]"
+                    ventral_bridge_width_mm = measure_pd[col_name_ventral][idx_row]
+                    if col_name_ventral in measure_pd.columns and not pd.isna(ventral_bridge_width_mm):
+                        axes[idx_row, idx_col].text(max(np.where(slice_lesion)[0]) + 3,
+                                                    min(np.where(slice_lesion)[1]),
+                                                    f'Ventral bridge\n{np.round(ventral_bridge_width_mm, 2)} mm',
+                                                    color='red', fontsize=12, ha='left', va='bottom')
+        # tight layout
+        mpl_plt.tight_layout()
+        for fname in imgs_to_generate.values():
+            mpl_plt.savefig(fname)
+        mpl_plt.close()
```

Comment:
What does this `close()` do? I don't think we ever open a figure window? (Since I saw a `close()` call I instinctively tried to look up whether there was a pyplot context manager, but I only got more confused.)

Reply:
good point. `close()` indeed seems be unnecessary. Maybe it stayed here from the developing phase when I opened the figure for debugging.

--> removing `close()`